lib/Conversion/ImportVerilog/Expressions.cpp

```diff
@@ -125,7 +120,8 @@ struct ExprVisitor {
     if (!preValue)
       return {};
     preValue = builder.create<moore::ReadLValueOp>(loc, preValue);
-    auto one = builder.create<moore::ConstantOp>(loc, preValue.getType(), 1);
+    auto one = builder.create<moore::ConstantOp>(
+        loc, preValue.getType().cast<moore::IntType>(), 1);
```

Comment:
I learned that `x.cast()/x.dyn_cast()/x.isa()` is being deprecated by llvm. Could we tweak it into `cast<moore::IntType>(preValue.getType())`?

Reply:
Oh yeah great catch, thanks!